.gitignore

```diff
@@ -1 +1,3 @@
 *.swp
+files/splunk
+files/universalforwarder
```

Comment:
The `.gitignore` file is managed by modulesync. If these files still need to be added to the `.gitignore` they should be added to the .sync.yml file.